README.md

```diff
@@ -1,22 +1,50 @@
-# Azure Kubernetes Service (AKS) Baseline Cluster
+# Azure Kubernetes Service (AKS) Multi Cluster
 
-This reference implementation demonstrates the _recommended starting (baseline) infrastructure architecture_ for a general purpose [AKS cluster](https://azure.microsoft.com/services/kubernetes-service). This implementation and document is meant to guide an interdisciplinary team or multiple distinct teams like networking, security and development through the process of getting this secure baseline infrastructure deployed and understanding the components of it.
+This reference implementation will go over some design decisions from the baseline to detail them as a well as incorporate some new _recommended infrastructure options_ for a Multi Cluster architecture. In this oportunity, this implementation and document are meant to guide the multiple distinct teams as introduced in the [AKS Baseline](https://github.com/mspnp/aks-secure-baseline) through the process of expanding from a single cluster to multiple clusters solution with a foundamental driver in mind which is **High Availability**.
 
-We walk through the deployment here in a rather _verbose_ method to help you understand each component of this cluster, ideally teaching you about each layer and providing you with the knowledge necessary to apply it to your workload.
+Throughout the reference implementation, you will see reference to _Contoso Bicycle_. They are a fictional small and fast-growing startup that provides online web services to its clientele on the east coast US-based. This narrative provides grounding for some implementation details, naming conventions, etc. You should adapt as you see fit.
+
+### Intro
+
+The app team works on a cluster strategically located in the `East US 2` region as this is where most of their customer base can be found. They have been operating this solo AKS cluster for a quite some time [following the recommended baseline practices to work with a single AKS cluster, and they learned them all by heart](https://github.com/mspnp/aks-secure-baseline).
+
+Thanks to baseline the cluster is already available from different _Zones_ within the same region, now they realize that provided `East US 2` went fully down, they are out of game. Even though the SLA(s) are acceptable for their business continuity plan, they are starting to think what are the options, and how their stateless application (Application ID: a0042) could increase its availability in case of an incident of the magnitude mentioned above. They started conversations with the business unit (BU0001) to increment the number of clusters by one. In other words, they are proposing to go over a Multi Cluster infrastructure solution in which multiple intances of the same application could live.
```

Comment:
```suggestion
The baseline the cluster was already available from different _Zones_ within the same region, but now they realize that if `East US 2` went fully down, zone coverage is not sufficient. Even though the SLA(s) are acceptable for their business continuity plan, they are starting to think what their options are, and how their stateless application (Application ID: a0042) could increase its availability in case of a complete regional outage. They started conversations with the business unit (BU0001) to increment the number of clusters by one. In other words, they are proposing to move to a multi-cluster infrastructure solution in which multiple instances of the same application could live.
```

Reply:
<3